graph_explorer/src/graph_explorer/graph_visualizer/module/content.py

```diff
@@ -34,16 +36,20 @@ def get_context(self):
             "current_visualizer": self.current_visualizer.get_name() if self.current_visualizer else None,
             "current_data_source": self.current_data_source.get_name() if self.current_data_source else None,
             "data_source_params": self.get_data_source_params(),
-            "content": self.current_visualizer.display(self.graph) if self.current_visualizer else None,
-            "workspaces": self.workspaces,
+            "content": self.current_visualizer.display(self.get_filtered_graph()) if self.current_visualizer else None,
+            "workspaces": [vars(ws) for ws in self.workspaces],
             "active_ws_id": self.workspace_id,
+            "filters": list(map(lambda filter: filter.to_json(), self.get_current_workspace().get_filters())) if self.workspace_id != -1 else []
```

Comment:
"self.workspace_id != -1"
Maybe we should create a const INVALID_WORKSPACE_ID = -1, so in the future we can easily understand what's going on